### PR TITLE
Expand instructions

### PR DESCRIPTION
**Pull request by ahepp**
An attempt at providing more clear instructions to install dependencies, per https://github.com/MabezDev/xtensa-rust-quickstart/issues/6

See the MD formatted: https://github.com/drewhepp/xtensa-rust-quickstart/tree/expand_instructions

### CODE REVIEW

**Comment by drewhepp** [comment]
I must confess: I didn't actually take these instructions for a test drive, since I didn't want to bother compiling llvm and rust again. Feedback appreciated.

File: README.md [hunk 1]
```diff
@@ -5,23 +5,63 @@
 
 ### Requirements
 
-    - xargo or cargo xbuild
-    - xtensa-esp32-elf toolchain must be in your `$PATH`
-    - esptool, if targeting the esp32 or esp8266
+#### llvm-xtensa
+Please refer to the [llvm-xtensa](https://github.com/espressif/llvm-xtensa) project for authoratative instructions.
```
Comment (MabezDev) [inline README.md]: That project is archived, https://github.com/espressif/llvm-project is the new one

Comment (ahepp) [reply README.md]: Thanks!

**Comment by MabezDev** [comment]
Thanks!

**Pull request merged**